tests/python_tests/test_vlm_pipeline.py

```diff
@@ -237,7 +237,7 @@ def test_vlm_continuous_batching_vs_stateful(config):
 
     models_path = get_ov_model(model_ids[0])
     for idx, links in enumerate(image_links_list):
-        stateful_pipe = VLMPipeline(models_path, "CPU", **get_default_llm_properties())
+        stateful_pipe = VLMPipeline(models_path, "CPU", ATTENTION_BACKEND="SDPA", **get_default_llm_properties())
```

Comment:
maybe we should also parametrize `test_vlm_pipeline` to have better per-model coverage?

And `test_vlm_with_scheduler_vs_default` seems redundant, as it compares PA vs PA